blog/_posts/2018-07-13-team-akaringular.md

```diff
@@ -0,0 +1,36 @@
+---
+title: New Tech Journey With RGSoC!
+layout:post
+created_at:Fri Jul 13 2018
+permalink: blog/2018-07-13-team-akaringular
+current: blog
+author:Rachael Kiteme
+twitter:rachaelkiteme
+categories: student-posts
```

Comment:
For the consistency purpose, please declare this as ' - student-posts' in the next line. You can check other files for the example.